### PR TITLE
Allow evenement/evenement 3.0 version

### PR DESCRIPTION
**Pull request by alexander-schranz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 language: php
+dist: trusty
```
Comment (alexander-schranz) [inline .travis.yml]: is needed to test php 5.5 on travis as the new dist does not support 5.5

**Comment by alexander-schranz** [comment]
@jens1o possible that you or another maintainer have some time to have a look at this and #33?

**Comment by jens1o** [comment]
(reopening to enqueue it for travis once again)

**Comment by jens1o** [comment]
Monolog has been removed and this pull request has been merged. I just wonder whether I forgot something before the release?

**Comment by alexander-schranz** [comment]
@jens1o thank you for merging. The symfony upgrade #40.

**Pull request merged**